### ISSUE TITLE
Tile Collision Editor - Cannot make objects visible after hiding them

### ISSUE THREAD

**Issue by Seanba** [bjorn/tiled#1929]
Hi Bjorn. I sometimes make objects in the tile collision editor invisible (mostly for testing) and I noticed there is no way to select these objects once they are hidden so I can make them visible again.

For now, I just edit the TSX file by hand so no big deal but thought you'd like to address this eventually all the same. Thanks.

**Comment by bjorn** [comment]
@X-Ray-Jin The problem in the Tile Collision Editor is that it does not have an Objects view. Hiding the objects is naturally available in the Properties view, but once deselected there is indeed on way to access these objects anymore.

> I sometimes make objects in the tile collision editor invisible (mostly for testing)

Do you think you could live without this feature? Cause one way of fixing this would be to avoid showing the "Visible" checkbox in the Properties when an object from the Tile Collision Editor is selected.

On the other hand, we'll have the same problem when introducing locking of individual objects, since they would no longer be selectable either. It would be unfortunate to forfeit this feature in the Tile Collision Editor as well, so maybe we should indeed look for a mode where interaction with invisible / locked objects is possible.

**Comment by X-Ray-Jin** [comment]
Oh actually i didnt recognize he was talking about the collision editor :P But still I guess it is good thing to consider. :+1:

**Comment by Seanba** [comment]
@bjorn Yeah, sure, removing the visible checkbox seems like a quick and easy way to fix this.

**Comment by bjorn** [comment]
> Yeah, sure, removing the visible checkbox seems like a quick and easy way to fix this.

It has been done. Thanks for bringing up this issue!
